src/ui/public/timepicker/relative_panel/kbn_timepicker_relative_panel.html

```diff
@@ -0,0 +1,139 @@
+<form role="form" ng-submit="applyRelative()" class="form-inline" name="relativeTime">
+  <div class="kbn-timepicker-body kuiVerticalRhythm">
+    <div class="kbn-timepicker-section kbn-timepicker-body-column">
+      <div class="kuiLocalDropdownHeader kbn-timepicker-form-header">
+        <label class="kuiLocalDropdownHeader__label">
+          From
+        </label>
+
+        <div class="kuiLocalDropdownHeader__actions">
+          <a
+            class="kuiLocalDropdownHeader__action"
+            ng-click="setRelativeToNow({key:'from'})"
+            kbn-accessible-click
+          >
+            Set To Now
+          </a>
+        </div>
+      </div>
+
+      <div class="kuiLocalDropdownFormNote kuiVerticalRhythmSmall">
+        <span ng-show="relative.from.preview">
+          {{relative.from.preview}}
+        </span>
+        <span ng-hide="relative.from.preview">
+          <em>Invalid Expression</em>
+        </span>
+      </div>
+
+      <div class="kuiFieldGroup kuiVerticalRhythmSmall">
+        <div class="kuiFieldGroupSection kuiFieldGroupSection--wide" ng-class="{ 'has-error': checkRelative() }">
+          <input
+            required
+            ng-model="relative.from.count"
+            ng-change="formatRelative({key:'from'})"
+            greater-than="-1"
+            min="0"
+            type="number"
+            class="form-control fullWidth">
+        </div>
+
+        <div class="kuiFieldGroupSection" ng-class="{ 'has-error': checkRelative() }">
+          <select
+            ng-model="relative.from.unit"
+            ng-options="opt.value as opt.text for opt in relativeOptions"
+            ng-change="formatRelative({key:'from'})"
+            class="form-control fullWidth">
+          </select>
+        </div>
+      </div>
+
+      <div class="kuiVerticalRhythmSmall">
+        <label>
```

Comment:
Can we use the `kuiCheckBox` class here, with the relevant markup for making sure the label is clickable. In the UI Framework, the example looks like this:

```html
<label class="kuiCheckBoxLabel">
  <input
    class="kuiCheckBox"
    type="checkbox"
  >
  <span class="kuiCheckBoxLabel__text">
    With clickable label
  </span>
</label>
```